### ISSUE TITLE
No quick fix available for mismatch return types

### ISSUE THREAD

**Issue by dulajdilshan** [ballerina-platform/ballerina-lang#29574]
**Description:**
No quick fix available for mismatch return types in resource functions

code snippet:
```ballerina
import ballerina/graphql;

service graphql:Service /graphql on new graphql:Listener(9090) {

    resource function get greetMe(string name) returns int {
        return "Sss";
    }
}
```
![Peek 2021-03-24 13-45](https://user-images.githubusercontent.com/25485997/112277246-8ee91880-8ca7-11eb-903f-d572a60a3d0e.gif)

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by IMS94** [comment]
@dulajdilshan This should also be fixed by ##29337. In SLA3, we had this issue when the method is within a class or service.

**Comment by nadeeshaan** [comment]
Closing based on the previous comment
